addons/packages/vsphere-cpi/1.21.0/bundle/sample-values/sample-value.yaml

```diff
@@ -0,0 +1,31 @@
+#@data/values
+#@overlay/match-child-defaults missing_ok=True
```

Comment:
It would be better to rename sample-value.yaml to vsphere_cpi.yaml, the way we do in downstream